build/build.xml

```diff
@@ -277,6 +277,28 @@
 			</else>
 		</if>
 
+		<!-- Checkout latest viglink to ext folder -->
+		<available file="${dir}/ext" type="dir" property="add-viglink-ext" />
+		<if>
+			<and>
+				<equals arg1="${add-viglink-ext}" arg2="1" trim="true" />
+				<equals arg1="${revision}" arg2="HEAD" trim="true" />
+			</and>
+			<then>
+				<exec dir="${dir}/ext" command="mkdir phpbb" passthru="true" />
+
+				<exec dir="${dir}/ext/phpbb" command="git clone https://github.com/phpbb-extensions/viglink.git viglink" passthru="true" checkreturn="true" />
+				<exec dir="${dir}/ext/phpbb/viglink" command="git checkout master" />
```

Comment:
Hmm, maybe we should checkout tagged release versions instead? Since development progress will probably take place on master.